wazuh/wazuh-dashboard-plugins
# Unification the rendering of flyout columns

### Description

The rendering values of the Mitre and Security operations flyout columns are changed to the same rendering of the events table, as it is the same rendering and can be handled all from the same place. 
And the rendering of the security operations requirements column is fixed.
 
### Issues Resolved

- #6844

### Evidence

Security operations

<img width="1508" alt="image" src="https://github.com/user-attachments/assets/9d09c520-39cc-4f1e-9180-aa84adb41cca">
<img width="1508" alt="image" src="https://github.com/user-attachments/assets/f17fcda3-29f3-48ab-8fbe-7538d9ee98df">
<img width="1506" alt="image" src="https://github.com/user-attachments/assets/65df60af-f10e-4de8-a9cd-45fb469a96d1">
<img width="1511" alt="image" src="https://github.com/user-attachments/assets/10e983bc-a9bc-4653-b5d4-3f7f4f1f17f2">
<img width="1508" alt="image" src="https://github.com/user-attachments/assets/7b7006ba-abd8-4b9e-8d79-6f85ec5fc207">

Mitre

<img width="1509" alt="image" src="https://github.com/user-attachments/assets/b643cf33-63b2-45ec-b241-65f5dbb3e04d">


### Test

1. Navigate to the different modules of Security operations > Control
2. Open flyouts 
3. Observe the following parameters:
- The agent.id and agent.name are links if they are not the agent 000 that redirects to the agent view
- The requirements if more than 1 must be separated by , 
- the rules must be a link that redirects to the rule.

1. Navigate to Mitre > Framework
2. Open flyout
3. Take into account the following parameters
- The agent.id and agent.name are links if they are not agent 000 that redirect to agent view
- The requirements if they are more than 1 must be separated by , 
- the rules must be a link that redirects to the rule.
- The technique must be a link that redirects to the technique.

1. Navigate to sameApplication > Events
2. Check that the links in the table work

1. Navigate to Threat Hunting > Dashboard
2. Check that the links in the table work: 
- Agent.id
- Agent.name
- rule.mitre.id
- rule.id

### Check List
